### PR TITLE
Add MPC notebook

### PR DESCRIPTION
**Pull request by barpit20**
I'm currently working on resolving an issue with the animated plots for this notebook. The contents can otherwise be reviewed.

### CODE REVIEW

File: script/modelling/model_predictive_control.jl [hunk 1]
```diff
@@ -0,0 +1,141 @@
+#' ---
+#' title: Model-Predictive Control
+#' ---
+
+#' **Originally Contributed by**: Robin Deits
+
+#' Let's implement a very simple model-predictive control (MPC) optimization in JuMP. 
+#' Specifically, we'll write an optimization that tries to find a sequence of control inputs
```
Comment (joaquimg) [inline script/modelling/model_predictive_control.jl:8]: "an optimization problem"

File: script/modelling/model_predictive_control.jl [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+#' ---
+
+#' **Originally Contributed by**: Robin Deits
+
+#' Let's implement a very simple model-predictive control (MPC) optimization in JuMP. 
+#' Specifically, we'll write an optimization that tries to find a sequence of control inputs 
+#' for a very simple model robot in order to optimize an objective function. 
+
+#' More concretely, our model will be a brick, sliding frictionlessly in two dimensions. 
+#' Our input will be the acceleration of the brick, and we'll try to minimize the brick's final velocity and distance from the origin.
```
Comment (joaquimg) [inline script/modelling/model_predictive_control.jl:12]: break long line

File: script/modelling/model_predictive_control.jl [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+# The gif() function saves our animated plot to an animated gif
+# Note: this may require you to have the `ffmpeg` program installed.
+# On Ubuntu 16.04, you can get this with `sudo apt-get install ffmpeg`.
+gif(anim, "img/mpc1.gif", fps = 30)
+
+#' ## Running the MPC Controller 
+#' In a real application, we wouldn't just run the MPC optimization once. 
+#' Instead, we might run the optimization at every time step using the robot's current state. 
+#' To do that, let's wrap the MPC problem in a function called `run_mpc()` that takes the robot's current position and velocity as input:
```
Comment (joaquimg) [inline script/modelling/model_predictive_control.jl:79]: break long line

**Review by joaquimg — COMMENTED** [review]
looks good

**Comment by matbesancon** [comment]
bump @barpit20 :)

**Comment by barpit20** [comment]
Sorry for the delay, I will fix this up soon.

**Comment by matbesancon** [comment]
bump :)

**Comment by odow** [comment]
Can documenter show gifs? This could be a cool tutorial to include.

**Pull request closed without merging**